### PR TITLE
Use <For /> in Status documentation

### PR DESCRIPTION
**Pull request by HT0323**
Closes https://github.com/yamada-ui/yamada-ui/issues/3774

## Description
Use the `<For />` component in `Status` docs

## Is this a breaking change (Yes/No):
No

### CODE REVIEW

File: docs/contents/components/feedback/status/index.mdx [hunk 1]
```diff
@@ -58,9 +68,31 @@ import { Status } from "@yamada-ui/react"
...
+        text: "Error",
+      },
+    ]}
+  >
+    {({ value, text }, index) => (
+      <Status key={index} value={value}>
+        {text}
+      </Status>
+    )}
+  </For>
```
Comment (Shunii85) [inline docs/contents/components/feedback/status/index.mdx:96]: You don't have to use `For` component. Sorry!! Please revert them.

**Review by Shunii85 — CHANGES_REQUESTED** [review]
@HT0323 Thanks for the PR !!
We took mistakes, this is why there are some request changes. I'm sorry!

**Comment by HT0323** [comment]
@Shunii85 
I've fixed it, please leave a review.

**Review by Shunii85 — APPROVED** [review]
@HT0323 
Thanks for the changes ! 
I'm sorry I'm late for re-review. LGTM !!

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**